cli/azd/pkg/tools/python/python.go

```diff
@@ -119,13 +128,25 @@ func (cli *PythonCli) CreateVirtualEnv(ctx context.Context, workingDir, name str
 	return nil
 }
 
-func pythonExe() string {
+func checkPath() (string, bool, error) {
```

Comment:
```suggestion
func checkPath() (pyString string, found bool, err error) {
```

Reply:
Makes it easier to read what the return values are, especially for 3 return values.